channels/test/liveserver.py

```diff
@@ -0,0 +1,205 @@
+import multiprocessing
+import os
+import sys
+
+import django
+from daphne.server import Server
+from django.core.exceptions import ImproperlyConfigured
+from django.test.testcases import TransactionTestCase
+from django.test.utils import modify_settings, override_settings
+from django.utils import six
+
+from .. import DEFAULT_CHANNEL_LAYER
+from ..asgi import ChannelLayerManager
+from ..worker import Worker, WorkerGroup
+
+# NOTE: We use ChannelLayerManager to prevent layer instance sharing
+# between forked process.  Some layers implementations create
+# connections inside the __init__ method.  After forking child
+# processes can lose the ability to use this connection and typically
+# stuck on some network operation.  To prevent this we use new
+# ChannelLayerManager each time we want to initiate default layer.
+# This gives us guaranty that new layer instance will be created and
+# new connection will be established.
```

Comment:
I'm not sure about this change. This commend doesn't change module usage in any case. It is a reminder to the person who will change this module in the future.